testsuite/features/secondary/srv_power_management.feature

```diff
@@ -82,6 +82,17 @@ Feature: Power management
     And I should see a "Power Off" button
     And I should see a "Reboot" button
 
+  Scenario: Cleanup: reset IPMI values
```

Comment:
Just a little nitpick. Can you make scenario name unique, please? Apart of this LGTM.

Reply:
fixed